src/figuresClasses.ts

```diff
@@ -1,19 +1,100 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape;
 
-}
+  isValidTriangle(): boolean {
```

Comment:
If the method has a name `isValidTriangle` - we should return true if it's valid

Reply:
Let's make the private method, cause you are using it only internally